timesketch/frontend/src/components/AppNavbarSecondary.vue

```diff
@@ -67,6 +67,12 @@ limitations under the License.
                   <span>Stories</span>
                 </router-link>
               </li>
+              <li v-bind:class="{ 'is-active': currentPage === 'sigma' }">
+                <router-link :to="{ name: 'SigmaOverview' }">
+                  <span class="icon is-small"><i class="fas fa-book" aria-hidden="true"></i></span>
```

Comment:
fa-book will collide with the Stories tab, right?

Reply:
yes, let me use another one